FPGA/core_rrv/de10_lite_core_rrv.qsf

```diff
@@ -490,8 +497,10 @@ set_global_assignment -name VERILOG_FILE mafia_adc/synthesis/submodules/DE10_Lit
 set_global_assignment -name VERILOG_FILE mafia_adc/synthesis/submodules/fiftyfivenm_adcblock_primitive_wrapper.v
 set_global_assignment -name VERILOG_FILE mafia_adc/synthesis/submodules/fiftyfivenm_adcblock_top_wrapper.v
 set_global_assignment -name VERILOG_FILE mafia_adc/synthesis/submodules/mafia_adc_adc_mega_0.v
+set_global_assignment -name SYSTEMVERILOG_FILE ../../source/common/macros.sv
```

Comment:
This seems wrong.
We don't need to compile macros file.
This is included and need to found in the search path